### ISSUE TITLE
[ Removed ]

### ISSUE THREAD

**Issue by ConnorAhern** [Prodigy-Hacking/ProdigyMathGameHacking#595]
[ Removed ]

**Comment by NameIsA** [comment]
Also, make blogs on the PMGH website about things PMGH is doing so that they appear in search results, getting more attention.

**Comment by ArcerionDev** [comment]
maybe

**Comment by ArcerionDev** [comment]
but if someone looks up "prodigy hacks" we're the first result so ehhhh

**Comment by NameIsA** [comment]
yes we are but people seem confused by the repository's "code" section because they dont think to scroll down 

thats what i most commonly hear when i tell someone about this repository

**Comment by NameIsA** [comment]
and we really need to do something about all the big prodigy YouTubers flexing about how they're hackers then not linking this repo- if they linked this repo more, it would get more traffic

**Comment by PatheticMustan** [comment]
a blog would be really cool!

**Comment by BoredFishRE** [comment]
I could do it

**Comment by NameIsA** [comment]
> I could do it

mk ty

**Comment by PatheticMustan** [comment]
https://prodigy-hacking.fandom.com/f

**Comment by NameIsA** [comment]
i mean, fandom isnt really for blogs, and doesnt fit the structure for a blog considering fandom wikis are supposed to be community-run

**Comment by NameIsA** [comment]
try google blogger

**Comment by irhrhd** [comment]
Or blogger.com

**Comment by NameIsA** [comment]
> Or blogger.com

that is what i said

**Comment by BoredFishRE** [comment]
i mean if we use blogger we're gonna seem like we're in our early 40s lmao. If anything we would use Ghost

**Comment by NameIsA** [comment]
> i mean if we use blogger we're gonna seem like we're in our early 40s lmao. If anything we would use Ghost

okie that sounds cool

doesnt matter to me what software you use whats important is just seo, design, and ease of use- this is to make everything as simple as possible for the script kiddies

**Comment by PatheticMustan** [comment]
roll our own blogging system >:D

**Comment by NameIsA** [comment]
I mean it would be really cool to do, but it would require a lot of work ngl

**Comment by NameIsA** [comment]
> i mean if we use blogger we're gonna seem like we're in our early 40s lmao. If anything we would use Ghost

i dont wanna use ghost because the UI would be kinda hard to do tbh

**Comment by NameIsA** [comment]
we could make an interface similar to github for familarity but it's just like how to install blog discussions 

i attached an image of something i designed with inspect element

![image](https://user-images.githubusercontent.com/61570792/109899328-8b371880-7c4a-11eb-830f-5dbe8c1f3965.png)

**Comment by NameIsA** [comment]
again, the goal is simplicity

while we're the first result on google, when skids click on it all they see is a bunch of code and it doesnt make as much sense

if the first result was instructions, then a place that is clearly a way to ask questions, then a blog for news about prodigyhacking, they would get more info

ofc we would still have the github for issue tracking, coding, etc. but the information source would be a website

**Comment by NameIsA** [comment]
going to be testing on this site https://dev.digitalteal.com/ because i have a website and why not test on a cool and fast and DOWNLOAD JUMPBOUND WHEN IT'S RELEASED BY https://digitalteal.com website that you already have

**Comment by irhrhd** [comment]
Did you guys pay for that domain if not where can i get 1

**Comment by NameIsA** [comment]
> Did you guys pay for that domain if not where can i get 1

i paid for it for something else but im using a subdomain of it to help test some things made for this repository

**Comment by NameIsA** [comment]
https://dev.digitalteal.com/index.php/forums/ should there be a forum on there or should it just redirect to the repository

react with 👍 for yes or 👎 for no

**Comment by BoredFishRE** [comment]
> 
> Did you guys pay for that domain if not where can i get 1

yes. we pay for it.

**Comment by irhrhd** [comment]
> try google blogger and set up custom domains so it's like blog.prodigyhacking.com then fix the SEO and design of prodigyhacking.com to be similar to pmgh github readme/wiki without all the code that is probably confusing to skids

Why do you call them skids isnt it kids?

**Comment by NameIsA** [comment]
> > try google blogger and set up custom domains so it's like blog.prodigyhacking.com then fix the SEO and design of prodigyhacking.com to be similar to pmgh github readme/wiki without all the code that is probably confusing to skids
> 
> Why do you call them skids isnt it kids?

skids = script kiddies

**Comment by NameIsA** [comment]
bump

**Comment by NameIsA** [comment]
This has to be finished before #529 because #490 will no longer be finished if there are so many files in this repository.

**Comment by NameIsA** [comment]
@BoredFishRE rename that "EEEE.webp" file and 1.) convert it to a png then 2.) rename it to "ProdigyMathGameHackingUbuntu.png" for SEO

also add `title` and `alt` properties to all the images

make sure there is exactly one `h1` tag on each page

add canonical URL tags to each page

change the footer so that there isnt a space in the mailto: and it says "The Prodigy Hacking Organization's website is created by BoredFish. Please email support@prodigyhacking.com if you have any questions regarding the Prodigy Hacking Organization."

add a link in the navbar saying "How to install" and it links to the pmgh wiki page saying how to install

remove the "click here for a link" and please fix the grammar 
![image](https://user-images.githubusercontent.com/61570792/110173681-d15bbb80-7db3-11eb-8e08-c2e18eebdaa3.png)

remove will from here https://prodigyhacking.com/aboutus.html and remove the ".html" from URLs and rename that page to /prodigyhackingstaff and add `alt` and `title` properties to all the images on that page, rename so that they include "ProdigyHacking" in the name

create a sitemap.txt (or sitemap.xml if you wanna do that) and a robots.txt for each page

this could use some changes to include "prodigy hacking" or smth somewhere for SEO
![image](https://user-images.githubusercontent.com/61570792/110173876-21d31900-7db4-11eb-9097-c8ce642d45fa.png)

**Comment by BoredFishRE** [comment]
> remove the "click here for a link" 
> ![image](https://user-images.githubusercontent.com/61570792/110173681-d15bbb80-7db3-11eb-8e08-c2e18eebdaa3.png)

People are super dumb so I have to add it.

> remove will from here https://prodigyhacking.com/aboutus.html 

He just retired legit yesterday. Have some patience.

> create a sitemap.txt (or sitemap.xml if you wanna do that) 

I've already done it, please god look. https://prodigyhacking.com/sitemap.xml

> this could use some changes to include "prodigy hacking" or smth somewhere for SEO
> ![image](https://user-images.githubusercontent.com/61570792/110173876-21d31900-7db4-11eb-9097-c8ce642d45fa.png)

Don't have control over how that looks.

> convert github.webp to png and rename github.webp to prodigyhacking-github.png

webp is used for speed. Using PNGs makes the site a bit slower to load.

The site is gonna get a revamp, I've started on it, just haven't done it.

**Comment by NameIsA** [comment]
please set up SPF and https forwarding https://www.hardenize.com/report/prodigyhacking.com?summary

**Comment by NameIsA** [comment]
> He just retired legit yesterday. Have some patience.
oop sorry i didnt know when he retired

**Comment by NameIsA** [comment]
> Don't have control over how that looks.


o

**Comment by NameIsA** [comment]
> webp is used for speed. Using PNGs makes the site a bit slower to load.

ah okay i see

**Comment by NameIsA** [comment]
> People are super dumb so I have to add it.

hmm okay that makes sense, what if it was like "Click here to join the Prodigy Hacking Discord"? i read those things like "click here" without any additional information arent good for SEO

**Comment by NameIsA** [comment]
> I've already done it, please god look. https://prodigyhacking.com/sitemap.xml

o sorry

**Comment by BoredFishRE** [comment]
also smth like blog.prodigyhacking.com isn't called a custom domain it's called a subdomain

**Comment by NameIsA** [comment]
> also smth like blog.prodigyhacking.com isn't called a custom domain it's called a subdomain

i know, i just mean the setting in blogger.com is "custom domain" but you can choose a subdomain

**Comment by BoredFishRE** [comment]
i'll do everything once I finish the new one. It'll be better. The current one is the first site I ever did.

**Comment by NameIsA** [comment]
> i'll do everything once I finish the new one. It'll be better. The current one is the first site I ever did.

okie

there is no rush im just giving my suggestions

**Comment by NameIsA** [comment]
on the website i've been working on *cough* https://digitalteal.com visit or noot kidnaps you at 3 am *cough* i've set up a vars.php file that stores thing like the meta description, staff members, etc. so that the website dynamically updates them

oh speaking of which is there a meta description tag and all that?

**Comment by BoredFishRE** [comment]
if you wanna see an example of a much better site I did https://boredfish.dev

**Comment by NameIsA** [comment]
> if you wanna see an example of a much better site I did https://boredfish.dev

okie

**Comment by BoredFishRE** [comment]
![sdfggasd](https://user-images.githubusercontent.com/49598984/110175183-5bfde400-7dcf-11eb-9e26-32b2d9c5fab9.PNG)
what the heck is this

**Comment by NameIsA** [comment]
> ![sdfggasd](https://user-images.githubusercontent.com/49598984/110175183-5bfde400-7dcf-11eb-9e26-32b2d9c5fab9.PNG)
> what the heck is this

work in progress site kek

its not very good yet because i havent finished it

**Comment by BoredFishRE** [comment]
![dfsgdfgdfg](https://user-images.githubusercontent.com/49598984/110175262-7df76680-7dcf-11eb-8166-f838e6dc6834.PNG)
I can tell

**Comment by NameIsA** [comment]
> ![dfsgdfgdfg](https://user-images.githubusercontent.com/49598984/110175262-7df76680-7dcf-11eb-8166-f838e6dc6834.PNG)
> I can tell

lmao

**Comment by NameIsA** [comment]
here's my suggestion for what to make your meta description tag:
```
The Prodigy Hacking Organization is attempting to raise awareness of security issues in Prodigy Math Game without giving users of our exploits the ability to cause any major damage. Join The Prodigy Hacking Organization in attempting to secure day-to-day education games!
```
obv you dont have to use that exact one but thats an example of what should go in there

**Comment by NameIsA** [comment]
bump

**Comment by ChickenOnYT** [comment]
https://bit.ly/prettymuchfreebitcoin

**Comment by ChickenOnYT** [comment]
no specific milestone, as soon as possible

**Comment by BoredFishRE** [comment]
The milestones aren for PHEx, not this.

**Comment by BoredFishRE** [comment]
oops misread, disregard comment

**Comment by ChickenOnYT** [comment]
mk

**Comment by ChickenOnYT** [comment]
![image](https://user-images.githubusercontent.com/61570792/110963660-314be800-8307-11eb-8469-515edc338716.png)

**Comment by ChickenOnYT** [comment]
this issue is canceled the admins have overruled it whatever

**Comment by PatheticMustan** [comment]
sad, I was interested in this

**Comment by ChickenOnYT** [comment]
well "overruled" as in bored and yama just said no and nobody else was online

**Comment by ChickenOnYT** [comment]
bored says he might do it

**Comment by ChickenOnYT** [comment]
so im leaving this open for now. also even if we dont use the website to grow we might grow in other ways

**Comment by PatheticMustan** [comment]
alright, cool.
I think it ight be a good idea to make Prodigy Hacking more *general* then it is right now. Prodigy is a pretty niche thing, maybe a general programming/game server might be cool?

**Comment by ChickenOnYT** [comment]
> alright, cool.
> I think it ight be a good idea to make Prodigy Hacking more _general_ then it is right now. Prodigy is a pretty niche thing, maybe a general programming/game server might be cool?

yes i agree with that

**Comment by ChickenOnYT** [comment]
but dont let prodigy hacking die

**Comment by ChickenOnYT** [comment]
https://www.reddit.com/r/ProdigyGameExploits/ its possible to "adopt" a subreddit if you have enough karma and had an account for 30 days, could we adopt this subreddit and use it to gain more attention?

**Comment by BoredFishRE** [comment]
https://www.reddit.com/r/ProdigyMathHacking/ we have one, and let's be real. Almost all our audience don't use reddit.

**Comment by ChickenOnYT** [comment]
thats true but i'll still go and post on the dead subreddit just to get more attention to pmgh lol

also r/ProdigyGame has a lot of members so ey maybe

**Comment by BoredFishRE** [comment]
I'm on it, everyone there absolutely hates hacking

**Comment by ChickenOnYT** [comment]
ik that's pretty normal

**Comment by ChickenOnYT** [comment]
the pmgw used to hate hacking but i went and got them all to like it

**Comment by MATEMATIK-SOSYE** [comment]
There has been a _majour_ issue with my CURRENTLY-SELECTED pets.

**Comment by PatheticMustan** [comment]
make a new issue, @MATEMATIK-SOSYE

**Comment by MATEMATIK-SOSYE** [comment]
> make a new issue, @MATEMATIK-SOSYE 

Yes. I am still kind of me to this application. How do I do so again? Please help me out? I am doing this from my phone, just so you know.

**Comment by ChickenOnYT** [comment]
bump

**Comment by PatheticMustan** [comment]
I really really really really want a blog

**Comment by BoredFishRE** [comment]
We can do that, just ask lmao

**Comment by technobyteongithub** [comment]
lol

**Comment by SakuraKitsunee** [comment]
@PatheticMustan i can make a maintained blog for pretty much anything you want, only if you want me to of course!

**Comment by KriticScripts** [comment]
@ChickenOnYT wdym everybody stil doesn't know how to hack?  people still now the basics and stuff

**Comment by ChickenOnYT** [comment]
" wdym everybody stil doesn't know how to hack? people still now the basics
and stuff"

From personal observation, people on the Prodigy wiki still ask me how to
hack.

**Comment by ChickenOnYT** [comment]
Also, I'm not really active here anymore. Don't ping me for stuff.
